packages/blockchain-wallet-v4-frontend/src/scenes/BuySell/CoinifyCheckout/Content/OrderHistory/template.success.js

```diff
@@ -49,7 +49,16 @@ const OrderHistory = (props) => {
           <Text size='15px' weight={400}>
             <FormattedMessage id='scenes.buysell.coinifycheckout.trades.pending' defaultMessage='Pending Orders' />
           </Text>
-          <OrderHistoryTable trades={filter(isPending, trades)} conversion={conversion} handleFinishTrade={trade => finishTrade(trade)} handleDetailsClick={trade => showModal('CoinifyTradeDetails', { trade })} handleTradeCancel={cancelTrade} status={status} cancelTradeId={cancelTradeId} />
+          <OrderHistoryTable
+            trades={filter(isPending, trades)}
+            conversion={conversion}
+            handleFinishTrade={trade => finishTrade(trade)}
+            handleDetailsClick={trade => showModal('CoinifyTradeDetails', { trade })}
+            handleTradeCancel={cancelTrade}
+            status={status}
+            canTrade={canTrade.getOrElse(true)}
```

Comment:
Should we be defaulting canTrade to true? Also can we do this in the selector instead of the template?

Reply:
This deals with rendering the little garbage can and deleting a trade. The thought process here was that if something went wrong, I'd rather let the user try and fail than not be able to try at all. Not sure which is worse or if we have a UX stance on this. I'll move it over to a selector anyway.